### PR TITLE
Fixed place holder issues in the input section.

### PR DESCRIPTION
**Pull request by musakanneh**
Created a div and inserted a search icon the better position the placeholder in the input section.

### CODE REVIEW

**Comment by musakanneh** [comment]
Adding to the Trending section to the about us page now to complete the page.

File: src/components/About/AboutHeader.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+};
+</script>
+
+<style lang='scss' scoped>
+.header {
+  width: 100%;
+  height: auto;
+  padding-bottom: 8%;
+  background: url(../../assets/images/jumbotron-bg.png);
+    height: 100vh;
```
Comment (nellybaz) [inline src/components/About/AboutHeader.vue]: why do you have another heights here

File: src/components/About/HeadCard.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+};
+</script>
+
+
+<style lang='scss' scoped>
+.core-values h2 {
+  width: 393px;
+  height: 30px;
+  left: 110px;
+  top: 1106px;
```
Comment (nellybaz) [inline src/components/About/HeadCard.vue]: Is this responsive as you shrink the page?

File: src/components/About/HeadCard.vue [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  font-weight: bold;
+  font-size: 25px;
+  line-height: 30px;
+  display: flex;
+  align-items: center;
+  color: #404040;
+}
+.event-card {
+  position: relative;
+  top: -60px;
```
Comment (nellybaz) [inline src/components/About/HeadCard.vue]: Is this responsive as you shrink the page?

File: src/components/About/HeadCard.vue [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    color: #6a6a6a;
+    line-height: 20px;
+    font-size: 14px;
+  }
+  .inner-content2 h2 {
+    border: 1px solid #6a6a6a1c;
+    padding: 27px 20px;
+    border-radius: 10px;
+    height: 20px;
+    top: 1198px;
```
Comment (nellybaz) [inline src/components/About/HeadCard.vue]: Is this responsive?

File: src/router/index.js [hunk 5]
```diff
@@ -11,16 +13,18 @@ const routes = [
...
-  //   // which is lazy-loaded when the route is visited.
-  //   component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
-  // }
+  {
+    path: '/about', 
+    component: About,
+  },
+  {
+    path: '/blog', 
+    component: Blog,
```
Comment (nellybaz) [inline src/router/index.js]: Use lazy loading to display this route

File: src/views/Blog.vue [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    PopularPosts,
+    RecentPosts
+  }
+};
+</script>
+
+
+
+<style scoped>
+@import url("https://rsms.me/inter/inter.css");
```
Comment (nellybaz) [inline src/views/Blog.vue]: Why do you need this import here

**Pull request merged**